netprofile_tickets/netprofile_tickets/views.py

```diff
@@ -795,14 +796,44 @@ def _send_ticket_mail(req, ticket=None, change=None):
 
     if change and change.transition:
         state = change.transition.to_state
+        old_state = change.transition.from_state
     else:
         state = ticket.state
+        old_state = None
 
     subject = '[T#%d] %s' % (ticket.id, ticket.name)
     recipient_map = {}
+    assigned_subscriptions = []
+
+    sess = DBSession()
+    with sess.no_autoflush:
+        if ticket.assigned_user and user_setting(
+                ticket.assigned_user,
+                'tickets.sub.notify_on_assign'):
+
+            tsub = UserTicketSubscription()
+            tsub.user = ticket.assigned_user
+            tsub.notify_change = True
+            tsub.ticket = ticket
+            sess.expunge(tsub)
+            assigned_subscriptions.append(tsub)
+
+        if ticket.assigned_group:
+            for user in ticket.assigned_group:
```

Comment:
Might be a better idea to use GroupTicketSubscription here instead of this loop.

Reply:
How to check user settings in this case?